src/service/resources-impl.js

```diff
@@ -686,9 +686,8 @@ export class Resources {
 
     if (this.documentReady_ && this.firstPassAfterDocumentReady_) {
       this.firstPassAfterDocumentReady_ = false;
-      this.viewer_.postDocumentReady(this.viewport_.getSize().width,
-        this.win.document.body./*OK*/scrollHeight);
-      this.updateScrollHeight_();
```

Comment:
Are you talking specifically for `#updateScrollHeight_`? If we're want to continue sending the `documentResized` event, we can do so without the `#updateScrollHeight_` (which is very badly broken right now).

```js
this.viewer_.postDocumentReady(this.viewport_.getScrollWidth(),
  this.viewport_.getScrollHeight());
// Do the important part of `#updateScrollHeight_`
this.viewer_.postDocumentResized(this.viewport_.getScrollWidth(),
  this.viewport_.getScrollHeight());
```